doc/rst/technotes/libraries.rst

```diff
@@ -591,3 +740,52 @@ supported.  We hope to extend this support in the future.
 
 C interoperability does not support default values for function arguments.  We
 do not anticipate supporting argument default values in C.
+
+.. _Python Intents:
+
+Intents in Python Interoperability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Libraries compiled for Python do not support the default intent for ``string``,
+``c_string`` or 1D arrays, as copies are currently performed (which would
+violate ``const ref``).  Instead, only the ``in`` and ``const in`` intents are
+supported for these types.  We may be able to support all intents in the future.
```

Comment:
Are there any work-arounds we could suggest to the user?

Should users declare their string, c_string, and 1D array args as `in` or `const in` in the function signatures? If so, it wouldn't hurt to be more explicit about it here.

Reply:
> Are there any work-arounds we could suggest to the user?

The work-arounds will vary for the unsupported intents and depending on how the user wants to use the argument with the unsupported intent, so I'm not sure it's worth being explicit about strategies.

> Should users declare their string, c_string, and 1D array args as in or const in in the function signatures? If so, it wouldn't hurt to be more explicit about it here.

Yup, I'll do that!

Reply:
> The work-arounds will vary for the unsupported intents and depending on how the user wants to use the argument with the unsupported intent, so I'm not sure it's worth being explicit about strategies.

OK, I suspected this might be the case. Seems like a level of detail more suitable for a specific python/chapel interop guide.